webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/resources/ProductPageResource.java

```diff
@@ -17,143 +17,148 @@
 import org.apache.oodt.cas.filemgr.structs.Reference;
 import org.apache.oodt.cas.metadata.Metadata;
 
+import javax.xml.bind.annotation.*;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * A JAX-RS resource representing a {@link ProductPage}.
- *
  * @author ngimhana (Nadeeshan Gimhana)
  */
-@XmlRootElement(name = "productPage")
-@XmlType(
-    propOrder = {
-      "pageSize",
-      "pageNum",
-      "totalPages",
-      "numOfHits",
-      "metadataResource",
-      "productResources"
-    })
+@XmlRootElement(name="productPage")
+@XmlType(propOrder = {"pageSize","pageNum","totalPages","numOfHits","metadataResource", "productResources"})
 @XmlAccessorType(XmlAccessType.NONE)
-public class ProductPageResource {
-
-  private MetadataResource metadataResource;
-  private List<ProductResource> productResources = new ArrayList<ProductResource>();
-  private int pageSize;
-  private int pageNum;
-  private int totalPages;
-  private long numOfHits;
-
-  /* The file manager's working directory for this resource, used for example
-   when creating zip archives.
-  */
-  private File workingDir;
-
-  /** Default constructor required by JAXB. */
-  public ProductPageResource() {}
-
-  /**
-   * Constructor that sets the ProductPage, metadata and working directory for the ProductPage
-   * resource.
-   *
-   * @param page the productpage for the dataset
-   * @param productMetaDataList the metadata for the dataset
-   * @param productReferencesList the References for the dataset
-   * @param workingDir the working directory for creating temporary files to attach to responses
-   */
-  public ProductPageResource(
-      ProductPage page,
-      List<Metadata> productMetaDataList,
-      List<List<Reference>> productReferencesList,
-      File workingDir) {
-
-    this.pageSize = page.getPageSize();
-    this.pageNum = page.getPageNum();
-    this.totalPages = page.getTotalPages();
-    this.numOfHits = page.getNumOfHits();
-
-    this.workingDir = workingDir;
-
-    List<Product> pageProducts = page.getPageProducts();
-    for (int i = 0; i < pageProducts.size(); i++) {
-
-      Product product = pageProducts.get(i);
-      Metadata metadata = productMetaDataList.get(i);
-      List<Reference> references = productReferencesList.get(i);
-
-      this.productResources.add(
-          new ProductResource(product, metadata, productReferencesList.get(0), workingDir));
+public class ProductPageResource
+{
+
+    private MetadataResource metadataResource;
+    private List<ProductResource> productResources = new ArrayList<ProductResource>();
+    private int pageSize;
+    private int pageNum;
+    private int totalPages;
+    private long numOfHits;
+
+
+    /* The file manager's working directory for this resource, used for example
+     when creating zip archives.
+    */
+    private File workingDir;
+
+    /**
+     * Default constructor required by JAXB.
+     */
+    public ProductPageResource()
```

Comment:
Opening curly bracket seems not formatted correctly. Should be inline with constructor.